libs/client/cornflow_client/core/application.py

```diff
@@ -188,7 +188,7 @@ def solve(
             .keys()
         )
         instance_errors = instance_checks.kfilter(lambda k: k not in warnings_tables)
-        if any([len(error_table) for error_table in instance_errors.values()]):
+        if len(instance_errors) > 0:
             log = dict(
```

Comment:
I think the old formulation, `any([len(error_table) for error_table in instance_errors.values()])`, was better, because it didn't generate an error when the check() method returned empty lists for some checks (for example: `{"check_error_foreign_keys: []}`).  I've seen that in InstanceCore.launch_all_checks, you only return the check results if the list is not empty, but some other projects might still override the check() method and return empty lists, and I think CornFlow should be able to handle this possibility

Reply:
You are right, i will change it back.